### ISSUE TITLE
RaXMX

### ISSUE THREAD

**Issue by Anaphory** [Anaphory/simuling#45]
Try reconstruction using RaXML, maybe: http://sco.h-its.org/exelixis/software.html

**Comment by LinguList** [comment]
Hi, short question: you want to start testing reconstruction? Then I'd run the full package: BEAST, MrBayes, etc., but we should make a list of potential coding types, and also potential settings, e.g.:

1 basic (=sample) vs. non-basic (200 concepts. vs. 500 etc.)
2 cross-semantic vs. binary coding, binary coding with specific rates for meaning sets, multi-state (=mk model, tested this in MrBayes, works basically, but only for < 10 languages)

This could take some time, but it may be a good idea, yet it could also be something for a future study.

**Comment by Anaphory** [comment]
Nope, not yet. I want to get the simulation out and reviewed first.
